pyiohat/parsers/ident/comet_2020_01_4_parser.py

```diff
@@ -31,6 +31,12 @@ def __init__(self, *args, **kwargs):
                 "header_translations"
             ]["translated_value"].items()
         }
+        self.metadata = {
+            "File Origin": "Comet",
```

Comment:
should we not capture the version? Or list of versions that produce a given pyiohat result ... what do you think?